fwoa-tools/package.json

```diff
@@ -38,10 +38,11 @@
     "testRegex": ".test.ts",
     "transform": {
       "\\.(ts)$": "ts-jest"
-    }
+    },
+    "testEnvironment": "node"
   },
   "dependencies": {
-    "axios": "0.25.0",
+    "axios": "0.26.0",
```

Comment:
I was getting this error `error: Error: Provided config url is not valid`

It was fixed in version `0.26`.
https://github.com/axios/axios/issues/4407